Dockerfile

```
@@ -0,0 +1,24 @@
+FROM python:3.13-slim
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)